client/src/components/search.js

```diff
@@ -0,0 +1,71 @@
+import React,{Component} from 'react';
+import SearchBox from './searchbox';
+import '../stylesheets/css/index.css';
+import '../stylesheets/css/components/App.css'
+import InputSearchService from '../services/api';
+
+class SearchComponent extends Component{
+    constructor(props){
+        super(props)
+        this.state = {
+            placeholderText: 'Enter a member name..',
+            searchType: 'member',
+            inputValue: '',
+            searchResults: InputSearchService.fetchSearch('members')
+        }
+        this.setPlaceholder = this.setPlaceholder.bind(this);
+        this.updateInputValue = this.updateInputValue.bind(this);
+    }
+
+//FUNCTION TO BE FIRED ON SELECTOR CHANGE
```

Comment:
Javascript has a block comment syntax that should be used for any method/function description. When you run something like jsdoc these will be pulled into the documentation automatically. It's the standard.

```
/**
* some comments
* some other comments
*/
```